.travis.yml

```diff
@@ -4,7 +4,7 @@ rvm:
   - 2.0
   - 2.1
   - 2.2
-  - 2.3.0-preview2
+  - 2.3.0
```

Comment:
Strangely, there doesn't appear to be a "short" alias for 2.3 in RVM, so I just used 2.3.0 instead.